app/src/main/res/values-it/strings.xml

```diff
@@ -40,9 +40,9 @@
 
     <!-- Force Update Alert Titel -->
     <string name="force_update_title">"Aggiorna"</string>
-    <string name="homescreen_handshakes_module_count_info_label">"I contatti saranno memorizzati in modo anonimo sul dispositivo"</string>
+    <string name="homescreen_handshakes_module_count_info_label">"Le co-presenze saranno memorizzate in modo anonimo sul dispositivo"</string>
     <string name="homescreen_handshakes_module_count_label_format">"{count} nelle ultime 48 ore"</string>
-    <string name="homescreen_handshakes_module_info_label">"Qui troverai le informazioni sulle persone incontrate che forniscono aggiornamenti sulla COVID-19."</string>
+    <string name="homescreen_handshakes_module_info_label">"Qui troverai le informazioni sulle persone con chi sei stato in co-presenza e che forniscono aggiornamenti sulla COVID-19."</string>
```

Comment:
sarebbe meglio "con cui sei stato in"

Reply:
co-presenza è più corretto ma, a mio avviso, meno leggibile.

Reply:
@dcremonini fatto.

Reply:
> co-presenza è più corretto ma, a mio avviso, meno leggibile.

Chiedo call diretta. Please dimmi quando posso chiamarti

Reply:
> > co-presenza è più corretto ma, a mio avviso, meno leggibile.
> 
> Chiedo call diretta. Please dimmi quando posso chiamarti

Anche subito come ci sentiamo?